Add WaveShare 5.5inch HDMI AMOLED mod for Voron 2.4

sttts opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod
    and/or other Voron users. Make sure to add information about how the mod was tested below.
  * [x] The mod is not merely a slight modification of an official Voron part, configuration
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile
    which only modifies a few values). *(When in doubt, contact one of the admins in the
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?

printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED

#### How was it tested?

Printed and used on my 2.4.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

![Attached to 2.4](https://github.com/VoronDesign/VoronUsers/blob/9d82caf60cf0f6922af265f1d0c1819ff6f7212f/printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/Images/photo.jpeg?raw=true)

#### Further notes

KiloQubit commented on printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/README.md:
```diff
@@ -0,0 +1,19 @@
+Required hardware
+-----------------
+
+In addition to things you can reuse from the stock build and to the skrew included in the display, these items are
```
Replace `skrew` with `screw`.

sttts replied:
done

KiloQubit commented on printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/README.md:
```diff
@@ -0,0 +1,19 @@
+
+This replaces the default Klipper display with a [Waveshare 5.5inch HDMI AMOLED display](https://www.waveshare.com/wiki/5.5inch_HDMI_AMOLED) as a skirt to the Voron 2.4.
+
+As the cover around the display itself has pretty tight fit and ABS shrinks 1-2%, you might have to print it 1% or 2% bigger.
```
All Voron parts and mods need to be designed so the STLs can be printed as is.  If printing the included STL has fitment issues, then please increase the size of the part appropriately so that it can be printed as is.

sttts replied:
done

KiloQubit reviewed (changes requested):
Thanks for your mod!

- Please remove spaces from your filenames.
- The display cover doesn't seem printable without supports.  If supports are required, they must be included in the STL, along with instructions to remove them after printing.
- The mount is not in a printable orientation.  What orientation did you print it in?  Did you have to use supports?  If so, supports must be included in the STL as above.

sttts commented:
> Please remove spaces from your filenames.
done
> The display cover doesn't seem printable without supports. If supports are required, they must be included in the STL, along with instructions to remove them after printing.

Added a solid back.

> The mount is not in a printable orientation. What orientation did you print it in? Did you have to use supports? If so, supports must be included in the STL as above.

fixed

KiloQubit commented:
PLEASE don't force push after a review, it makes it very hard to see what changed since the review was done.

sttts commented:
> PLEASE don't force push after a review, it makes it very hard to see what changed since the review was done.

@KiloQubit just click there:

![image](https://user-images.githubusercontent.com/730123/114284085-baad2200-9a4d-11eb-9a8e-d3c776257f69.png)

KiloQubit reviewed (changes requested):
Thanks for the updates!  It looks like the part of the holes in the mount print in air:

<img width="436" alt="Screen Shot 2021-04-10 at 16 55 20" src="https://user-images.githubusercontent.com/69389593/114284412-ae10d580-9a1d-11eb-8fed-578857de4c59.png">

sttts commented:
> It looks like part of the holes in the mount print in air

I thought the slicer is clever enough to create bridges. I can make it 45°.

KiloQubit reviewed (approved):
Thanks for the updates!
